nautobot/core/views/generic.py

```diff
@@ -295,7 +295,7 @@ def post(self, request, *args, **kwargs):
 
             try:
                 with transaction.atomic():
-                    object_created = form.instance.pk is None
+                    object_created = not form.instance.present_in_database
```

Comment:
This raises the question to me as to whether we might consider this property returning `None` if it's not `True`? Would that be more intuitive since people have become so used to `instance.pk is None` as a sentinel value for "this object is not in the database"?

Reply:
To me returning `True or None` is less intuitive than returning `True or False`. I'll see if I can find a good place to add this pattern to the docs though.